dashboard/src/utils/data/extract-nodes.ts

```diff
@@ -12,7 +12,7 @@ export function extractNodes(wfSpec: WfSpec, threadSpec: ThreadSpec): CustomNode
       type: 'node',
       data: {
         node,
-        type: getNodeType(node),
+        type: getNodeType(node).type,
```

Comment:
Why is this now `getNodeType(node).type` the purpose of the function is to get the nodeType, what is this returning if not the type directly?